src/lib.rs

```diff
@@ -138,17 +146,13 @@
 //! between these three coordinate systems, although you should still strive to handle this, as
 //! they're still relevant in more niche area such as Mac Catalyst, or multi-tasking on tablets.
 //!
-//! There is, however, a different important concept: The "safe area". This can be accessed with
-//! [`Window::safe_area`], and describes a rectangle in the surface that is not obscured by notches,
-//! the status bar, and so on. You should be drawing your background and non-important content on
-//! the entire surface, but restrict important content (such as interactable UIs, text, etc.) to
-//! being drawn in the safe area.
+//! Note that the safe area (discussed above) is especially important here, since windows on mobile
+//! are often full screen, and often the device has notches (as illustrated in the image below).
```

Comment:
Still not sure how it applies to coordinate systems, since it doesn't change anything about them, because safe area doesn't alter how they work in any way and just says that you have _void areas_ in your _rendering_, but coordinate systems are more about directions.

Reply:
How about:
```suggestion
//! This is illustrated in the image below, along with the safe area since it's often relevant on
//! mobile.
```

?

Reply:
I guess it's fine.